linux_os/guide/system/network/network-kernel/network_host_parameters/sysctl_net_ipv4_ip_forward/rule.yml

```diff
@@ -40,6 +40,12 @@ ocil: |-
     {{{ ocil_sysctl_option_value(sysctl="net.ipv4.ip_forward", value="0") }}}
     The ability to forward packets is only appropriate for routers.
 
+warnings:
+    - functionality: |-
+        Certain technologies such as virtual machines, containers, etc. rely on IPv4 forwarding to enable and use networking.
+        Disabling IPv4 forwarding would cause those technologies to stop working. Therefore, this rule should not be used in
+        profiles or benchmarks that target usage of IPv4 forwarding.
```

Comment:
Thanks for creating this warning. Should be very helpful for those tailoring or justifying dropping this rule locally based on how the endpoint is being used.